packages/protocol-validation/package.json

```diff
@@ -0,0 +1,43 @@
+{
+	"name": "@codaco/protocol-validation",
+	"version": "3.1.0-alpha.1",
```

Comment:
think this should be v4.0.0

Reply:
bumped to 4.0.0 [b3d22d0](https://github.com/complexdatacollective/network-canvas-monorepo/pull/78/commits/b3d22d029ab2ad19a0d340d03b4748e6ee8b67d3)